concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1786,6 +1822,24 @@ jobs:
                 ruby paas-cf/concourse/scripts/extract_tf_vars_from_terraform_state.rb \
                 < vpc-tfstate/vpc.tfstate > terraform-variables/vpc.tfvars.sh
 
+      - task: disable-bosh-resurrector
```

Comment:
We should consider how and when to re-enable the resurrector

Reply:
Yeah you can see up above I've had some thoughts on it. Thought it might be a bit of a footgun having the `enable-az` jobs blindly turn it back on. I think the answer might be alerting - should I create a story for that do you think?

Reply:
I'm not convinced that `bosh deploy` doesn't turn it back on to be honest, so we might be having a moot conversation. We should check it.

Reply:
But also yes, let's create a story for the alerting. IMO we're better off getting this out there and then expanding it, over trying to get everything in place. An availability zone could go away tomorrow.